package.json

```diff
@@ -50,32 +58,121 @@
         "path": "./syntaxes/markdown.tmLanguage.json"
       }
     ],
-    "themes": [
```

Comment:
was the `themes` section deleted accidentally?

Reply:
On purpose, it was causing problems since it wasn't fully functional and it's meant for full blown VSCode themes, which I didn't think was the intention. If you want the preview to follow the current theme, that's doable, but is a larger work item and may involve some coordination with the d2 exe.